server/fleet/traits.go

```diff
@@ -4,12 +4,12 @@ import "time"
 
 // Createable contains common timestamp fields indicating create time
 type CreateTimestamp struct {
-	CreatedAt time.Time `json:"created_at" db:"created_at"`
+	CreatedAt time.Time `json:"created_at" db:"created_at" goqu:"skipupdate"`
 }
 
 // UpdateTimestamp contains a timestamp that is set whenever an entity is changed
 type UpdateTimestamp struct {
-	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
+	UpdatedAt time.Time `json:"updated_at" db:"updated_at" goqu:"skipupdate"`
```

Comment:
Got a feeling that won't scale super well, e.g. if we rely on it to be updatable, but later on someone implements another update and adds the `skipupdate` tag... I mean sure, a test should fail, but that could become annoying quickly. We'll see, I guess.

Reply:
We could definitely say "let's only use this for annoying inserts/updates/deletes, and for complex logic in select, but nothing else". @lucasmrod thoughts?

Reply:
+1, this is not looking good, possible bugs + the reader has to go to the struct definitions...

Reply:
Fiiiine, moving it to the old way :)